feature/feature.go

```diff
@@ -65,6 +65,11 @@ const (
 
 	// BootstrapFormatIgnition will allow an user to enable alternate machine bootstrap format, viz. Ignition.
 	BootstrapFormatIgnition featuregate.Feature = "BootstrapFormatIgnition"
+
+	// ExternalResourceGC is used to enable the garbage collection of external resources like NLB/ALB on deletion
+	// owner: @richardcase
+	// alpha: v1.4
```

Comment:
Unless we plan to backport it to v1.4
```suggestion
	// alpha: v1.5
```

Reply:
Good spot :+1: